tcms/testplans/models.py

```diff
@@ -92,6 +94,9 @@ def _get_absolute_url(self):
     def get_absolute_url(self):
         return self._get_absolute_url()
 
+    def get_full_url(self):
+        return super().get_full_url().rstrip('/')
```

Comment:
We need a test for that b/c it is a modified behavior on which other parts of the application will rely. The test must assure this URL doesn't contain `/` at the end.

Also try using self.client.get() with the resulting URL - ideally (that is if the test client behaves like a normal browser) when there is a trailing / it will fail and when there isn't it will load the TP page.